SKILL.tsv

```diff
@@ -5531,8 +5531,8 @@ SKILL_20151102_005530	{#DD5500}{ol}[Missile]{/}{/}{nl}Fires a stone bullet to pu
 SKILL_20151102_005531	{#DD5500}{ol}[Missile]{/}{/}{nl}Load multiple arrows into your crossbow and then fire them in quick succession at an enemy.
 SKILL_20151102_005532	{#DD5500}{ol}[Missile] - [Poison]{/}{/}{nl}Blows a toxic arrow that inflicts damage over a long period of time.
 SKILL_20151102_005533	{#DD5500}{ol}[Missile] - [Poison]{/}{/}{nl}Fires a contagious poison arrow. Poisoned target spreads poison around to nearby enemies whenever it is attacked.
-SKILL_20151102_005534	{#DD5500}{ol}[Missile]{/}{/}{nl}Fires a noise-making arrow. Enemies near the target fall under fear.
-SKILL_20151102_005535	{#DD5500}{ol}[Missile]{/}{/}{nl}Shoot a flare at a designated area. Upon hitting the ground, the flare bursts into sparks that damages nearby enemies.
+SKILL_20151102_005534	{#DD5500}{ol}[Missile]{/}{/}{nl}Fires a nois-making arrow to a target. Enemies nearby the target will be afflicted with Fear.
+SKILL_20151102_005535	{#DD5500}{ol}[Missile]{/}{/}{nl}Fires a flare on the ground at a targeted area. Upon hitting the ground, the flare bursts into sparks that damages nearby enemies.
```

Comment:
To stay consistent with the other descriptions, we may want to change instances of "Fires" to "Fire," since other descriptions have been altered to not have things like "Tempers a weapon" or "Attacks an enemy"